blocks/adaptors/classification_output.py

```diff
@@ -2,18 +2,26 @@
 
 from .base import Adaptor
 from type import DataType, PredsWithProbs
-from typing import List
+from typing import List, Optional
 
 
 class ClassificationOutputAdaptor(Adaptor):
     inputTypes = DataType.PredictionsWithProbs
     outputType = DataType.List
 
-    def __init__(self, select: int):
+    def __init__(self, select: int, dict_lookup: Optional[dict] = None):
```

Comment:
what's a dict lookup? I don't really get this

Reply:
can we rather move this out of ClassificationOutputAdaptor and deal with it inside Huggingface's model? we shouldn't pollute the rest of the blocks with HF specific stuff - it should emit "PredsWithProbs" as defined in types.

Reply:
Yes, Sorry this remained in ClassificationOutputAdaptor but I have actually already moved it to HF. Deleting this now.

Reply:
Fixed it.